Doc/library/asyncio-eventloop.rst

```diff
@@ -186,19 +186,25 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
+   the threads in the :class:`~concurrent.futures.ThreadPoolExecutor`.
+   After calling this method,
+   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
    while using the default executor.
 
-   The *timeout* parameter specifies the amount of time the executor will
-   be given to finish joining. The default value is ``None``, which means the
-   executor will be given an unlimited amount of time.
+   The *timeout* parameter specifies the duration in seconds (as a :class:`float`)
+   this method gives the executor to finish joining.
```

Comment:
```suggestion
   The *timeout* parameter specifies the amount of time
   (in :class:`float` seconds) the executor will be given to finish joining.
```

I had trouble getting active voice to not sound awkward here, and I don't think it really adds much to reader understanding here—as a parameter description in an the API reference, the method being described is the clear actor in this case. Therefore, I reverted to the original wording, with the key clarification that the time given is in (floating point) seconds.

Reply:
Float seconds is critical.  I might replace 'will be given' to 'will have', which removes the passive.

Reply:
We might, but the current framing is consistent with this being an external, discretionary timeout acting on the executor, as opposed to a requirement intrinsic to the executor or its context.